cli.js

```diff
@@ -9,8 +9,33 @@ const debounce = require('lodash.debounce');
 const hasAnsi = require('has-ansi');
 const mem = require('mem');
 const clipboardy = require('clipboardy');
+const inquirer = require('inquirer');
+
```

Comment:
I will update the PR when I'm on the computer again. Maybe there's more to review... I can make a review commit then. 😅